girder_annotation/girder_large_image_annotation/web_client/views/imageViewerWidget/geojs.js

```diff
@@ -129,7 +129,6 @@ var GeojsImageViewerWidgetExtension = function (viewer) {
          */
         _addPixelmapLayerParams(layerParams, pixelmapElement) {
             // For pixelmap overlays, there are additional parameters to set
-            layerParams.keepLower = false;
```

Comment:
I think we need this set to false, since otherwise you'll get off effects if the superpixels are transparent.

I think this was fixed in https://github.com/OpenGeoscience/geojs/commit/7ea49772a89031353aacacc2387e33f7e0bf14a6.  Are you on the master build of geojs?

Reply:
`npm` is telling me that I have `geojs@1.7.1`, which looks like the latest release.

With `layerParams.keepLower = false`, I was seeing a problem in HisotmicsUI where zooming in past the number of levels would cause the pixelmap to disappear until zoomed out again. For example, in my testing I was working with a whole slide image with magnification 20, so HUI allows zooming in to 40x. The pixelmap layer would not be visible when zooming between 20x and 40x.

Without setting `keepLower = false`, I did not see that issue. I didn't see any effects (not sure what I should be looking out for though). Is there a better way to allow this kind of zooming while keeping the layer visible?

Reply:
You should obviously see the issue if the fill color is transparent and the stroke color is opaque.  We either need to change some parameters or there is a bug in GeoJS.  I'll investigate.

Reply:
There will be a PR in GeoJS shortly.

Reply:
I see the issue now. I'll go ahead and add that line back. I'm still not sure how to deal with the issue of zooming "past" the pixelmap layer that I'm seeing in HistomicsUI.

Reply:
https://github.com/OpenGeoscience/geojs/pull/1178.  I'll merge when it passes CI and then it will be available.

Reply:
Looks like that PR did the trick. I've added `keepLower = false` back in 65ae579